src/main/java/com/streamr/client/dataunion/EthereumTransactionReceipt.java

```diff
@@ -0,0 +1,18 @@
+package com.streamr.client.dataunion;
+
+import org.web3j.protocol.core.methods.response.TransactionReceipt;
+
+
+/**
+ * wrapper to avoid exposing web3j TransactionReceipt
```

Comment:
is this just hygiene or is there a more concrete technical reason? Maybe add to comment "because it's better style" or something

Reply:
@jtakalai Streamr client API should only expose Streamr classes.